Firestore/Swift/Source/SwiftAPI/Pipeline/PipelineSource.swift

```diff
@@ -13,44 +13,46 @@
 // limitations under the License.
 
 @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
-public struct PipelineSource: @unchecked Sendable {
+public struct PipelineSource<P>: @unchecked Sendable {
```

Comment:
Hi Andy, I remember in the discussion we decided to avoid using generic and go with `RealtimePipelineSource` ?